01-javascript/04-promise/anirudh_promise.js

```diff
@@ -0,0 +1,104 @@
+//Method 1
+const{readFile} = require('fs');
+const fsPromises = require('fs').promises;
+
+
+
+function readfiles()
+{
+  return new Promise((res, rej) => {
+    fsPromises.readFile('./index.js',{encoding:'utf-8'})
+          .then(data => {
+                 fsPromises.readFile('./anirudhsinghtomar.js',{encoding:'utf-8'})
+                    .then(datab => {
```

Comment:
format code please

Reply:
I have added the files mentioned in the code into the folder.....